README.md

```diff
@@ -0,0 +1,20 @@
+[![Hackage version](https://img.shields.io/hackage/v/test-framework.svg?label=Hackage&color=informational)](http://hackage.haskell.org/package/test-framework)
+[![test-framework on Stackage Nightly](https://stackage.org/package/test-framework/badge/nightly)](https://stackage.org/nightly/package/test-framework)
+[![Stackage LTS version](https://www.stackage.org/package/test-framework/badge/lts?label=Stackage)](https://www.stackage.org/package/test-framework)
+[![Cabal build](https://github.com/haskell/test-framework/workflows/Haskell-CI/badge.svg)](https://github.com/haskell/test-framework/actions)
+
+
+test-framework
+==============
+
+This framework for organize tests is considered legacy and no longer actively developed.
```

Comment:
```suggestion
This framework for organize tests is mature and stable, but it is no longer actively developed.

NEW FEATURE REQUESTS AND CONTRIBUTIONS ARE NO LONGER ACCEPTED!
```

Reply:
@andreasabel I'm OK with the old wording as well, just throwing this in as a suggestion..